saml/authn_request.go

```diff
@@ -45,10 +45,10 @@ func getAuthnRequestOptions(opt ...Option) authnRequestOptions {
 
 // AllowCreate is a Boolean value used to indicate whether the identity provider is allowed, in the course
 // of fulfilling the request, to create a new identifier to represent the principal.
-func AllowCreate() Option {
+func AllowCreate(allow bool) Option {
```

Comment:
Given this is a flag, we might come around it without a bool param. Something linke `DisableAllowCreate()`.